app/controllers/application_controller.rb

```diff
@@ -1,4 +1,20 @@
 class ApplicationController < ActionController::API
+  before_action :check_host
+
   include DeviseTokenAuth::Concerns::SetUserByToken
   include RenderErrors
+
+  PLAY_ENV = '"playground"'
+
+  def check_host
+    puts "Where is it from: " , request.headers['location']
+    if request.headers['location'].eql? PLAY_ENV
```

Comment:
Should not use request header to switch your database. Create a column, set a boolean to indicate production / non-production application.